app/endpoints/amap.py

```diff
@@ -0,0 +1,117 @@
+from fastapi import APIRouter
+
+router = APIRouter()
+
+
+@router.get("/products")
```

Comment:
we need to have `amap` on all those requests as the endpoint
```suggestion
@router.get("/amap/products")
```

Reply:
Adding prefix in include_router in api.py is better but same effect. Done